Tools/machines/karolina-it4i/karolina_gpu.qsub

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -l
+
+# Copyright 2023 The WarpX Community
+#
+# This file is part of WarpX.
+#
+# Authors: Axel Huebl, Andrei Berceanu
+# License: BSD-3-Clause-LBNL
+
+#PBS -q qgpu
+#PBS -N WarpX
+# use all eight GPUs per node
+#PBS -l select=1:ncpus=128:ngpus=8:mpiprocs=8,walltime=00:30:00
+#PBS -o WarpX.o$PBS_JOBID
+#PBS -e WarpX.e$PBS_JOBID
+#PBS -A <proj>
+
+# Define the WarpX output directory
+SCRATCHDIR=/scratch/project/${PBS_ACCOUNT,,}/${USER}/WarpX
+
+# Change to the local scratch directory, exit on failure
+cd /lscratch/${PBS_JOBID} || exit
+
+# executable & inputs file or python interpreter & PICMI script here
+EXE=./warpx.rz
+INPUTS=inputs_rz
+
+# OpenMP threads
+export OMP_NUM_THREADS=$(($PBS_NCPUS / $PBS_NP))
+
+# run
+mpirun -np ${PBS_NP} bash -c "
+    export CUDA_VISIBLE_DEVICES=\${MPI_LOCALRANKID};
```

Comment:
@ax3l, do you agree with the pinning solution offered by IT4I in [this gist](https://gist.github.com/berceanu/f9e8118c508ad9a23df1f7bb80c1c9a9#gpu_bindsh) based on `OMPI_COMM_WORLD_LOCAL_RANK`, or do we keep the one based on `MPI_LOCALRANKID` from here instead?

Reply:
That is ok as well! Only works with OpenMPI, but that is the MPI on the cluster, so let's go.
Will update.